scripts/token-hud-extension.js

```diff
@@ -1,4 +1,7 @@
-class TokenHudExtension {
+import WFRP_Utility from "../../../systems/wfrp4e/modules/system/utility-wfrp4e.js";
```

Comment:
So I'm not sure why the imports are needed. Additionally, I've noticed there were some issues with cross-module imports reported by Forge users. I found this conversation between KaKaRoTo, MooMan and LeRatierBretonnien from here useful. https://discord.com/channels/170995199584108546/697845419593695362/774409394586386483